index.html

```diff
@@ -14,45 +14,254 @@
-      <p id="intro">
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hesitate to contact me.
-      </p>
-      <p id="connect">LET'S CONNECT</p>
-      <ul class="social">
-        <li>
-          <a href="https://www.twitter.com" target="_self"
-            ><img src="tw.png" alt="twitter icon"
-          /></a>
-        </li>
-        <li>
-          <a href="https://www.linkedin.com" target="_self"
-            ><img src="linkedin.png" alt="linkedin icon"
-          /></a>
-        </li>
-        <li>
-          <a href="https://www.medium.com" target="_self"
-            ><img src="medium.png" alt="medium icon"
-          /></a>
-        </li>
-        <li>
-          <a href="https://www.github.com" target="_self"
-            ><img src="github.png" alt="github icon"
-          /></a>
-        </li>
-        <li>
-          <a href="https://www.vectorstock.com" target="_self"
-            ><img src="v.png" alt="vector stock icon"
-          /></a>
-        </li>
-      </ul>
-    </section>
+    <main>
+      <section id="headline">
+        <h1 id="greeting">I’m Andrew. Glad to see you!</h1>
+        <p class="intro">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <p class="connect">LET'S CONNECT</p>
+        <ul class="social">
+          <li>
+            <a href="https://www.twitter.com" target="_self"
+              ><img src="tw.png" alt="twitter icon"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com" target="_self"
+              ><img src="linkedin.png" alt="linkedin icon"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.medium.com" target="_self"
+              ><img src="medium.png" alt="medium icon"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.github.com" target="_self"
+              ><img src="github.png" alt="github icon"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.vectorstock.com" target="_self"
+              ><img src="v.png" alt="vector stock icon"
+            /></a>
+          </li>
+        </ul>
+      </section>
+      <section class="works_container">
+        <article>
+          <img
+            class="card_image"
+            src="Snapshoot Portfolio1.png"
+            alt="project1 image"
+          />
+          <h3 class="projectName_holder">Tonic</h3>
+
+          <div class="canopy">
+            <h4>CANOPY</h4>
+            <div></div>
+            <p>Back End Dev</p>
+            <div></div>
+            <p>2015</p>
```

Comment:
- [ ] please, like the previous reviewer noted, the dots between the card small title and date should be added. this is part of the original design, and we are expected to follow as such. Please, kindly add it.

yours
![Screenshot (1217)_LI](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/65298048/bfc0cceb-bc2e-4a87-922a-e73cd1e73f25)
original
![Screenshot (1218)_LI](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/65298048/3ec634db-acce-45ef-8b42-fce9d4969bbf)